src/cdk/a11y/key-manager/list-key-manager.spec.ts

```diff
@@ -467,6 +467,16 @@ describe('Key managers', () => {
         expect(keyManager.setActiveItem).toHaveBeenCalledWith(0);
       });
 
+      // This test should pass if all items are disabled and the down arrow key got pressed.
+      // If the test setup crashes or this test times out, this test can be considered as failed.
+      it('should not get into an infinite loop if all items are disabled', () => {
+        keyManager.withWrap();
+        keyManager.setActiveItem(0);
```

Comment:
You can chain the `withWrap` and `setActiveItem`.

Reply:
I tried to keep it consistent with the rest of the tests. All of those don't seem to chain either.